### ISSUE TITLE
vip.idlixofficial.net: detection

### ISSUE THREAD

**Issue by superview1** [uBlockOrigin/uAssets#23363]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://vip.idlixofficial.net/`

### Category

detection

### Description

the adblock detected

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/167547813/b94473e8-da82-4023-af99-9053e8778876)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.0
Chromium: 123
filterset (summary):
 network: 140264
 cosmetic: 77881
 scriptlet: 22560
 html: 0
listset (total-discarded, last-updated):
 added:
  IDN-0: 9910-2, 6m
  adguard-cookies: 29267-48, 6m
  ublock-cookies-adguard: 1079-1, 6m
 default:
  user-filters: 0-0, never
  easylist: 86090-567, 43m Δ
  easyprivacy: 50791-812, 43m Δ
  plowe-0: 3735-0, 6m
  ublock-badware: 8298-0, 43m Δ
  ublock-filters: 37550-322, 43m Δ
  ublock-privacy: 816-0, 43m Δ
  ublock-quick-fixes: 318-1, 43m Δ
  ublock-unbreak: 2252-3, 43m Δ
  urlhaus-1: 12505-0, 1h.4m
filterset (user): [empty]
trustedset:
 added: [array of 12 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 789 ms (selfie)
 maxAssetCacheWait: 293 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 4
 network:
  doubleclick.net: 1
  google-analytics.com: 1
  googlesyndication.com: 1
  googletagmanager.com: 1
 extended:
  ##img[alt^="QQ101" i]
  ##img[alt^="QQ724" i]
  ##img[alt^="QQ801" i]
  ##img[alt^="QQ820" i]
  ##img[alt^="QQ828" i]
  ##img[alt^="QQ882" i]
  ##.module_home_ads
```
</details>

**Comment by JobcenterTycoon** [comment]
https://github.com/uBlockOrigin/uAssets/issues/23361

**Comment by superview1** [comment]
> #23361

I apologize in advance, but the issue submitted has not been resolved why the case closed?

**Comment by JobcenterTycoon** [comment]
Its closed with a commit https://github.com/uBlockOrigin/uAssets/commit/5b6e2d57536aedc6b6d7931d9b5b13fdc269bdf7 Run https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-filters&manual=1 and test again.

**Comment by superview1** [comment]
> Its closed with a commit [5b6e2d5](https://github.com/uBlockOrigin/uAssets/commit/5b6e2d57536aedc6b6d7931d9b5b13fdc269bdf7) Run https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-filters&manual=1 and test again.

ohhh yes my bad I'm sorry, I just found out how to do it. Thank you very much.
